committee-steering/governance/sig-governance-requirements.md

```diff
@@ -0,0 +1,76 @@
+# SIG Governance Requirements
+
+## Goals
+
+This document outlines the recommendations and requirements for defining SIG and subproject governance.
+
+This doc uses [rfc2119](https://www.ietf.org/rfc/rfc2119.txt) to indicate keyword requirement levels.
+Sub elements of a list inherit the requirements of the parent by default unless overridden.
+
+## Checklist
+
+Following is the checklist of items that should be considered as part of defining governance for
+any subarea of the Kubernetes project.
+
+### Roles
+
+- *MUST* enumerate any roles within the SIG and the responsibilities of each
+- *MUST* define process for changing the membership of roles
+  - When and how new members are chosen / added to each role
+  - When and how existing members are retired from each role
+- *SHOULD* define restrictions / requirements for membership of roles
+- *MAY* define target staffing numbers of roles
+
+### Organizational management
+
+- *MUST* define when and how collaboration between members of the SIG is organized
+  - *SHOULD* define how periodic video conference meetings are arranged and run
+  - *SHOULD* define how conference / summit sessions are arranged
+  - *MAY* define periodic office hours on slack or video conference
+
+- *MAY* define process for new community members to contribute to the area
+  - e.g. read a contributing guide, show up at SIG meeting, message the google group
+
+### Project management
+
+- *MUST* define how subprojects are managed if at all
+  - When and how new subprojects are created
+  - Roles within subprojects
+- *MUST* define how priorities / commitments are managed
```

Comment:
I wonder whether this one would be better as "SHOULD"?

Reply:
ACK